sklearn/pipeline.py

```diff
@@ -697,6 +698,19 @@ def set_params(self, **kwargs):
         self._set_params('transformer_list', **kwargs)
         return self
 
+    def _check_params(self):
+        show_warning = False
+        for idx, (name, trans) in enumerate(self.transformer_list):
+            if trans is None:
+                self.transformer_list[idx] = (name, 'drop')
+                show_warning = True
+
+        if show_warning:
+            warnings.warn(
+                "Transformers set to None is now set with 'drop' "
+                "in version 0.20 and will be removed in 0.22.",
```

Comment:
I don't think this message is clear. Please reword it in a way that is helpful to someone who formerly used None